src/clj/dashboard_clj/routes.clj

```diff
@@ -23,15 +25,81 @@
   (GET "/version" _
     {:status 200
      :headers {"Content-Type" "text/json; charset=utf-8"}
-     :body (str (json/write-str {:version (version/get-version
-                                            "vanilla"
-                                            "vanilla"
-                                            "version number not found")}))})
+     :body (do
+             (prn "get version")
+             (str (json/write-str {:version (version/get-version
+                                              "vanilla"
+                                              "vanilla"
+                                              "version number not found")})))})
 
   (GET "/services" _
     {:status 200
      :headers {"Content-Type" "text/json; charset=utf-8"}
-     :body (str (json/write-str {:services (h/get-services)}))})
+     :body (do
+             ;(prn "get services")
+             (str (json/write-str {:services (h/get-services)})))})
+
+;;;;; Layout Saving Routes
+
+  ;TODO test removing the write-str here and put the de-string back in url handler
+  (GET "/layout" req
```

Comment:
we need to look into how to document/comment routes. we also need to look into how we can start to split this file up across multiple namespaces by semantic purpose (users/login, layout, data subscriptions, etc.)